### PR TITLE
Use the new module import path

### PR DESCRIPTION
**Pull request by jeffallen**

### CODE REVIEW

File: byzcoin/heartbeat_test.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func TestHeartbeat_Start(t *testing.T) {
 }
 
 func TestHeartbeat_Timeout(t *testing.T) {
-	t.Skip("https://github.com/dedis/cothority/issues/1473")
+	t.Skip("https://go.dedis.ch/cothority/v3/issues/1473")
```
Comment (kc1212) [inline byzcoin/heartbeat_test.go]: this should stay on github

File: doc.go [hunk 2]
```diff
@@ -11,7 +11,7 @@
 // protocols who make conodes work together.
 //
 // For further documentation, please go to
-// https://github.com/dedis/cothority/blob/master/README.md.
+// https://go.dedis.ch/cothority/v3/blob/master/README.md.
```
Comment (kc1212) [inline doc.go]: this should stay on github too

File: byzcoin/heartbeat_test.go (hunk 1 above)
Comment (Gilthoniel) [reply byzcoin/heartbeat_test.go]: I guess we should have a redirection anyway

Comment (jeffallen) [reply byzcoin/heartbeat_test.go]: The go,dedis.ch domain is just for the meta data that the go build tool uses. More redirects would just be more confusing.

**Comment by jeffallen** [comment]
A fix for the comments from @kc1212 are in #1679.

**Pull request merged**